modules/price/src/Form/CurrencyForm.php

```diff
@@ -65,6 +73,9 @@ public function form(array $form, FormStateInterface $form_state) {
     $form['numericCode'] = [
       '#type' => 'textfield',
       '#title' => $this->t('Numeric code'),
+      '#description' => $this->t('The numeric currency code. According to ISO 4217 this code consists of three digits 
```

Comment:
"The numeric currency code." is redundant, just repeats the element title. 
Maybe just "The three digit code as defined by ISO 4217."?